x/cork/client/cli/tx.go

```diff
@@ -169,3 +169,71 @@ Where proposal.json contains:
 
 	return cmd
 }
+
+// GetCmdSubmitScheduledCorkProposal implements the command to submit scheduled cork proposal
+func GetCmdSubmitScheduledCorkProposal() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "schedule-cork [proposal-file]",
+		Args:  cobra.ExactArgs(1),
+		Short: "Submit a scheduled cork proposal",
+		Long: strings.TrimSpace(
+			fmt.Sprintf(`Submit a scheduled cork proposal along with an initial deposit.
+The proposal details must be supplied via a JSON file.
+
+Example:
+$ %s tx gov submit-proposal schedule-cork <path/to/proposal.json> --from=<key_or_address>
+
+Where proposal.json contains:
+
+{
+  "title": "Dollary-doos LP Scheduled Cork Proposal",
+  "description": "I trust them, approve cork",
+  "block_height": 100000,
+  "target_contract_address": "0x123801a7D398351b8bE11C439e05C5B3259aeC9B",
+  "contract_call_proto_json": "{\"function_name\":\"cork\",\"function_args\":[]}",
```

Comment:
Similarly here, some more specific comments/references to the `contract_call_proto_json` are probably warranted.